rs/registry/admin/src/main.rs

```diff
@@ -3708,7 +3708,7 @@ async fn main() {
             eprintln!("Download IC-OS .. ");
             let tmp_dir = tempfile::tempdir().unwrap().into_path();
             let mut tmp_file = tmp_dir.clone();
-            tmp_file.push("temp.gz");
+            tmp_file.push("temp-image");
```

Comment:
Why is there now no file extension?

Reply:
Similarly to above, upgrades are agnostic to gz or zst upgrade images, so we are dropping the extension to reflect this. The code still works without the file extension 👍